demos/top-app-bar.html

```diff
@@ -238,6 +258,10 @@ <h3>Demo Controls</h3>
           appBar.destroy();
           appBar = mdc.topAppBar.MDCTopAppBar.attachTo(appBarEl);
         });
+
+        prominentCheckbox.addEventListener('change', function()  {
+          appBarEl.classList[this.checked ? 'add' : 'remove']('mdc-top-app-bar--prominent');
```

Comment:
I'd point out that we're disabling the prominent checkbox when short is checked, but not vice versa... do we want these to consistently either toggle each other or disable each other? (If we make it so that prominent disables short, we shouldn't need to uncheck prominent in the short checkbox handler since it would need to be unchecked manually first)

Reply:
Fixed. I changed it so the prominent disabled the short checkbox and vice versa.